src/Service/MailTemplate/ComplexMacroHelper.php

```diff
@@ -27,63 +36,90 @@ public function buildMacros($entity): array
+    private function buildCaseMacros(CaseEntity $case): array
+    {
+        // Note: Setting text on the link will break the link.
+        $values['case.link'] = new ComplexMacro(
+            $this->createLink(
+                $this->router->generate('case_show', ['id' => $case->getId()], RouterInterface::ABSOLUTE_URL),
+                $this->translator->trans('Open case in browser', [], 'case')
+            ),
+            'Link to the case'
+        );
 
-            // Board members
-            $table = new Table([
-                'unit' => TblWidth::TWIP,
-                'cellMargin' => 0,
-                'spacing' => 0,
-            ]);
+        return $values;
+    }
 
-            foreach ($entity->getBoardmembers() as $boardmember) {
-                $roles = $boardmember->getBoardRoles();
-                $this->addTableRow($table, [
-                    $roles->isEmpty()
-                        ? sprintf('%s', $boardmember->getName())
-                        : sprintf('%s (%s)', $boardmember->getName(), implode(', ', $roles->map(static fn (BoardRole $role) => $role->getTitle())->toArray())),
-                ]);
-            }
+    private function buildAgendaMacros(Agenda $agenda): array
+    {
+        // Note: Setting text on the link will break the link.
+        $values['agenda.link'] = new ComplexMacro(
+            $this->createLink(
+                $this->router->generate('agenda_show', ['id' => $agenda->getId()], RouterInterface::ABSOLUTE_URL),
+                $this->translator->trans('Open agenda in browser', [], 'agenda')
+            ),
+            'Link to the agenda'
+        );
+
+        // Agenda items
+        $table = new Table([
+            'unit' => TblWidth::TWIP,
+            'cellMargin' => 0,
+            'spacing' => 0,
+        ]);
+
+        foreach ($agenda->getAgendaItems() as $agendaItem) {
+            $this->addTableRow($table, [
+                sprintf('%s–%s',
+                    $agendaItem->getStartTime()->format('H:i'),
+                    $agendaItem->getEndTime()->format('H:i')
+                ),
+                $agendaItem->getTitle(),
+                $agendaItem->getMeetingPoint(),
+            ]);
+        }
 
-            $values['agenda.board_members'] = new ComplexMacro($table, 'Board members');
+        $values['agenda.items'] = new ComplexMacro($table, 'Agenda items');
+
+        // Board members
+        $table = new Table([
+            'unit' => TblWidth::TWIP,
+            'cellMargin' => 0,
+            'spacing' => 0,
+        ]);
+
+        foreach ($agenda->getBoardmembers() as $boardmember) {
+            $roles = $boardmember->getBoardRoles();
+            $this->addTableRow($table, [
+                $roles->isEmpty()
+                    ? sprintf('%s', $boardmember->getName())
+                    : sprintf('%s (%s)', $boardmember->getName(), implode(', ', $roles->map(static fn (BoardRole $role) => $role->getTitle())->toArray())),
```

Comment:
If a board member has roles on two different boards this would display both. We probably only want to display the role he has on the relevant board.